src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -932,6 +932,121 @@ public static void GetFullPath_Windows_Wildcards(char wildcard)
+            yield return new object[] { "foo", @"\\.\C:\Foo2\..", @"\\.\C:\foo" };
+
+            // Local host paths
+            yield return new object[] { "foo", @"\\LOCALHOST\share1", @"\\LOCALHOST\share1\foo" };
+            yield return new object[] { "foo", @"\\LOCALHOST\share2", @"\\LOCALHOST\share2\foo" };
+            yield return new object[] { "foo", @"\\LOCALHOST\share3\dir", @"\\LOCALHOST\share3\dir\foo" };
+            yield return new object[] { "foo", @"\\LOCALHOST\share4", @"\\LOCALHOST\share4\foo" };
+            yield return new object[] { "foo", @"\\LOCALHOST\share5", @"\\LOCALHOST\share5\foo" };
+            yield return new object[] { "foo", @"\\LOCALHOST\share6\", @"\\LOCALHOST\share6\foo" };
+            yield return new object[] { "foo", @"\\LOCALHOST\  share7\", @"\\LOCALHOST\  share7\foo" };
+            yield return new object[] { "foo", @"\\?\UNC\LOCALHOST\share8\test.txt.~SS", @"\\?\UNC\LOCALHOST\share8\test.txt.~SS\foo" };
+            yield return new object[] { "foo", @"\\?\UNC\LOCALHOST\share9", @"\\?\UNC\LOCALHOST\share9\foo" };
+            yield return new object[] { "foo", @"\\?\UNC\LOCALHOST\shareA\dir", @"\\?\UNC\LOCALHOST\shareA\dir\foo" };
+            yield return new object[] { "foo", @"\\?\UNC\LOCALHOST\shareB\. ", @"\\?\UNC\LOCALHOST\shareB\. \foo" };
+            yield return new object[] { "foo", @"\\?\UNC\LOCALHOST\shareC\.. ", @"\\?\UNC\LOCALHOST\shareC\.. \foo" };
+            yield return new object[] { "foo", @"\\?\UNC\LOCALHOST\shareD\    ", @"\\?\UNC\LOCALHOST\shareD\    \foo" };
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\  shareE\", @"\\.\UNC\LOCALHOST\  shareE\foo" };
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\shareF\test.txt.~SS", @"\\.\UNC\LOCALHOST\shareF\test.txt.~SS\foo" };
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\shareG", @"\\.\UNC\LOCALHOST\shareG\foo" };
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\shareH\dir", @"\\.\UNC\LOCALHOST\shareH\dir\foo" };
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\shareK\", @"\\.\UNC\LOCALHOST\shareK\foo" };
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\  shareL\", @"\\.\UNC\LOCALHOST\  shareL\foo" };
+
+            // Current drive rooted
+            yield return new object[] { @"\tmp\bar", curDir, Path.GetPathRoot(curDir)+ @"tmp\bar" };
+            yield return new object[] { @"\.\bar", curDir, Path.GetPathRoot(curDir) + @"bar" };
+            yield return new object[] { @"\tmp\..", curDir, Path.GetPathRoot(curDir).Substring(0,2)};
+            yield return new object[] { @"\tmp\bar\..", curDir, Path.GetPathRoot(curDir) + @"tmp" };
+            yield return new object[] { @"\tmp\bar\..", curDir, Path.GetPathRoot(curDir) + @"tmp" };
+            yield return new object[] { @"\", curDir, Path.GetPathRoot(curDir) };
+
+            // Specific drive Rooted
```

Comment:
nit consistent casing Rooted->rooted

The reason consistency is good is it makes it easier for the eye to catch inconsistency that matters.